### PR TITLE
Fix TensorRT-6 Version Parsing

### PR DESCRIPTION
**Pull request by jhalakpatel**
1. find_cuda_config.py - Fix version parsing for TRT 6.  If required version >= 6, parse version information from NvInferVersion.h, else parse version information from NvInfer.h.

### CODE REVIEW

**Comment by jhalakpatel** [comment]
@annarev Do you have any comments or suggestions on the review?

File: third_party/tensorrt/tensorrt_configure.bzl [hunk 1]
```diff
@@ -22,102 +22,96 @@ _TF_NEED_TENSORRT = "TF_NEED_TENSORRT"
...
+
+  # Set up BUILD file.
+  _tpl(repository_ctx, "BUILD", {
+      "%{copy_rules}": "\n".join(copy_rules),
+  })
 
 tensorrt_configure = repository_rule(
-    implementation = _tensorrt_configure_impl,
-    environ = [
+    implementation=_tensorrt_configure_impl,
```
Comment (aaroey) [inline third_party/tensorrt/tensorrt_configure.bzl]: Please revert these changes as well.

File: third_party/tensorrt/tensorrt_configure.bzl [hunk 2]
```diff
@@ -22,102 +22,96 @@ _TF_NEED_TENSORRT = "TF_NEED_TENSORRT"
...
-    _TF_TENSORRT_HEADERS = [
-        "NvInferVersion.h",
-        "NvInfer.h",
-        "NvUtils.h",
-        "NvInferPlugin.h",
-        "NvInferRTSafe.h",
-        "NvInferRTExt.h",
-        "NvInferPluginUtils.h",
-    ]
+  actual = [int(v) for v in actual_version.split(".")]
```
Comment (aaroey) [inline third_party/tensorrt/tensorrt_configure.bzl]: Please revert all indention changes in this file. .bzl files requires 4-space indention.

**Review by aaroey — CHANGES_REQUESTED** [review]

**Comment by jhalakpatel** [comment]
@aaroey Addressed review comments.

**Review by aaroey — APPROVED** [review]

**Pull request merged**